scripts/GenerateMessengerDatabase.groovy

```diff
@@ -0,0 +1,23 @@
+import org.codehaus.groovy.runtime.InvokerHelper
+class GenerateMessengerDatabase extends Script {
```

Comment:
I see lots of possible improvements but as I know this script is not part of our application I shall refrain from commenting xD